Workbooks/Azure Monitor - Workspaces/Agent Health/Agent Health.workbook

```diff
@@ -0,0 +1,615 @@
+              "formatOptions": {
+                "showIcon": true
+              }
+            },
+            {
+              "columnMatch": "Heartbeat Trend",
+              "formatter": 10,
+              "formatOptions": {
+                "palette": "redGreen",
+                "showIcon": true
+              },
+              "tooltipFormat": {
+                "tooltip": "Each bar represents the bucket of time based on the Unhealthy Criteria. Showing last 30 buckets max."
+              }
+            },
+            {
+              "columnMatch": "Details",
+              "formatter": 5,
+              "formatOptions": {
+                "linkTarget": "CellDetails",
+                "linkLabel": "🔍 View Details",
+                "linkIsContextBlade": true,
+                "showIcon": true
+              }
+            },
+            {
+              "columnMatch": "ResourceId",
+              "formatter": 0,
+              "formatOptions": {
+                "showIcon": true
+              }
+            },
+            {
+              "columnMatch": "Category",
+              "formatter": 0,
+              "formatOptions": {
+                "showIcon": true
+              }
+            },
+            {
+              "columnMatch": "OS Type",
+              "formatter": 0,
+              "formatOptions": {
+                "showIcon": true
+              }
+            },
+            {
+              "columnMatch": "Time Ago",
+              "formatter": 0,
+              "formatOptions": {
+                "showIcon": true
+              }
+            },
+            {
+              "columnMatch": "OSType",
+              "formatter": 0,
+              "formatOptions": {
+                "showIcon": true
+              }
+            },
+            {
+              "columnMatch": "ComputerEnvironment",
+              "formatter": 0,
+              "formatOptions": {
+                "showIcon": true
+              }
+            },
+            {
+              "columnMatch": "LastHeartbeat",
+              "formatter": 0,
+              "formatOptions": {
+                "showIcon": true
+              }
+            }
+          ],
+          "filter": true,
+          "labelSettings": [
+            {
+              "columnId": "_ComputerName_",
+              "label": "_ComputerName_"
+            },
+            {
+              "columnId": "Computer",
+              "label": "Computer"
+            },
+            {
+              "columnId": "State",
+              "label": "State"
+            },
+            {
+              "columnId": "Environment",
+              "label": "Environment"
+            },
+            {
+              "columnId": "OS",
+              "label": "OS"
+            },
+            {
+              "columnId": "Azure Resource",
+              "label": "Azure Resource"
+            },
+            {
+              "columnId": "Time",
+              "label": "Time"
```

Comment:
It would be more intuitive if this column named something like 'Last Heartbeat' instead of 'Time'